<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanelCA</class>
 <widget class="QWidget" name="ControlPanelCA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority Setting:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAutomatic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>133</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonManual">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>133</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>309</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="labelPSL">
      <property name="text">
       <string>Pressure Switch Low [bar]:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelPSLL_2">
      <property name="text">
       <string>Pressure Switch High [bar]:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBoxPSL">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>22</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBoxPSH">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="maximum">
       <number>35</number>
      </property>
      <property name="value">
       <number>25</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPSLL">
      <property name="text">
       <string>Pressure Switch Low Low [bar]:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBoxPSLL">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonOnCompressor0">
      <property name="text">
       <string>On</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonOffCompressor0">
      <property name="text">
       <string>Off</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonOffCompressor1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonOnCompressor1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>On</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelCompressor2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>139</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>Compressor #2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressor1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>139</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Compressor #1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
